controlsfx/src/test/java/org/controlsfx/control/textfield/CustomTextFieldTest.java

```diff
@@ -1,18 +1,18 @@
 /**
  * Copyright (c) 2019, ControlsFX
```

Comment:
Since you are modifying this file, add 2020 after 2019. It's better not to change the formatting of the copyright header so that they all look the same.

Reply:
done